l10n_it_ddt/views/stock_picking_package_preparation.xml

```diff
@@ -85,6 +85,9 @@
                        domain="['|', ('id',  '=', partner_id), ('parent_id', '=', partner_id)]"
                        attrs="{'invisible':[('ddt_type_id', '=', False)], 'required':[('ddt_type_id', '!=', False)]}"/>
             </field>
+            <field name="picking_ids" position="attributes">
+                <attribute name="widget">one2many_list</attribute>
+            </field>
```

Comment:
@SimoRubi this prevents the user to add existing available pickings in `Transfers` tab of DDT

Reply:
@eLBati verified, then the widget change cannot be a solution